src/group_theory/noncomm_pi_coprod.lean

```diff
@@ -1,6 +1,6 @@
 /-
 Copyright (c) 2022 Jocchim Breitner. All rights reserved.
-Released under Apache 2.0 license as described in the file LICENSE.
+Released under Apache 2.0 license as described in the file LιCENSE.
```

Comment:
Err

Reply:
🤦‍♂️

I should start using vim, I can't use vscode I guess

Reply:
Yes, I failed to use the right replace command. Will fix.